cypress/support/index.js

```diff
@@ -0,0 +1,17 @@
+import '@dhis2/cli-utils-cypress/support'
+import './commands'
+
+import { loginAndPersistSession } from './server'
+
+//enableNetworkShim()
```

Comment:
Remove all commented-out code in the file

Reply:
Removed